subprojects/distributions-dependencies/build.gradle.kts

```diff
@@ -141,7 +119,7 @@ dependencies {
         api(libs.cglib)                 { version { strictly("3.2.6") }}
         api(libs.equalsverifier)        { version { strictly("2.1.6") }}
         api(libs.flightrecorder)        { version { strictly("7.0.0-alpha01") }}
-        api(libs.guice)                 { version { strictly("2.0") }}
+        api(libs.guice)                 { version { require("2.0") }}
```

Comment:
Is this a leftover from testing?

Reply:
No. This is a test-only dependency and running tests in `build-init` actually needs a higher version now. But this is also used in another project (only for testing). But I can check if that other project can be upgraded to newer Guice. Then I can use strictly with the higher Guice version here an it is less confusing.

But I wonder if this global `strictly`s here for test-only dependencies should go. They don't really make sense, right?